Source/Core/Core/HW/GBAPadEmu.cpp

```diff
@@ -0,0 +1,115 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Core/HW/GBAPadEmu.h"
+
+#include "InputCommon/ControllerEmu/Control/Input.h"
+#include "InputCommon/ControllerEmu/ControlGroup/Buttons.h"
+#include "InputCommon/GCPadStatus.h"
+
+static const u16 dpad_bitmasks[] = {PAD_BUTTON_UP, PAD_BUTTON_DOWN, PAD_BUTTON_LEFT,
+                                    PAD_BUTTON_RIGHT};
+
+static const u16 button_bitmasks[] = {PAD_BUTTON_B,  PAD_BUTTON_A,  PAD_TRIGGER_L,
+                                      PAD_TRIGGER_R, PAD_TRIGGER_Z, PAD_BUTTON_START};
+
+static const char* const named_buttons[] = {"B", "A", "L", "R", "Select", "Start"};
+
+GBAPad::GBAPad(const unsigned int index) : m_reset_pending(false), m_index(index)
+{
+  // Buttons
+  groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
+  for (const char* named_button : named_buttons)
+  {
+    ControllerEmu::Translatability translate = ControllerEmu::DoNotTranslate;
+    std::string ui_name = named_button;
+
+    if (named_button == std::string("Start"))
+    {
+      translate = ControllerEmu::Translate;
+      ui_name = _trans("START");
```

Comment:
It would probably be simpler to just make the normal name uppercase as well. The reason why that isn't the case for GC controllers is because we didn't want to break compatibility with old INIs where it isn't in uppercase. Though I guess there is an argument to be made that it's nice to have GBA INIs look the same as GC INIs, so don't treat this as a required change.